include/lmms_math.h

```diff
@@ -34,11 +34,10 @@
 #include <numbers>
 #include <concepts>
 
-#include "lmmsconfig.h"
-#include "lmms_constants.h"
 
 namespace lmms
 {
+inline constexpr float F_EPSILON = 1.0e-10f; // 10^-10
```

Comment:
(Non-blocking) Why was this moved out of `lmms_constants.h`? That seems like a reasonable home for it.

Reply:
I was actually planning to remove that file at start, but realised we still need the constants file. But this change remained.

Reply:
Oops forgot this. I just turned off my laptop, will have to wait 2 days if i should fix this.

Reply:
done